dev-libs/libcss/libcss-0.5.0.recipe

```diff
@@ -50,7 +53,9 @@ INSTALL()
 {
 	make install PREFIX=$prefix NSSHARED=/system/data/netsurf-buildsystem \
 		INCLUDEDIR=$relativeIncludeDir
-	mkdir -p $developLibDir
+		
+	mkdir -p $libDir
```

Comment:
Instead of doing this (manual move of the libs), it would be better to tell make to install them to the right place directly:

    make install PREFIX=$prefix NSSHARED=/system/data/netsurf-buildsystem INCLUDEDIR=$relativeIncludeDir LIBDIR=$relativeLibDir